pkg/packagekit/updates.css

```diff
@@ -66,6 +66,34 @@ tr.security {
     border: none;
 }
 
+.flow-list-blank-slate {
+    padding-top: 8em;
+    margin: 0 auto;
+    max-width: 68rem;
+    text-align: center;
+}
+
+.flow-list {
+    margin: 1em 1em 3em;
+    padding: 0;
+    width: 100%;
+    max-width: 68rem;
+    text-align: left;
+    box-sizing: border-box;
+}
+
+.flow-list li {
+    text-align: left;
+    box-sizing: border-box;
+    width: 30ex;
```

Comment:
I did the suggested changes. That makes the wrapping more consistent, but still doesn't provide a centered layout (e. g. if you only have one or two packages) on the blank-slate pages ("no updates" or "reboot recommended"). But IMHO it's good enough for now.